keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -0,0 +1,1077 @@
+  non-trivial computations, excessive access to disks (including increased log
+  volume), significant amount of data sent and/or received over network, etc.
+  This through this both in small and large cases, again with respect to the
+  [supported limits].
+
+### Troubleshooting
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+
+* **What are other known failure modes?**
+  For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+
+* **What steps should be taken if SLOs are not being met to determine the problem?**
+
+[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
+
+## Optional Future Extensions
+
+The following features have been considered for future extensions of the this proposal. They are out
+of scope for the initial proposal and/or implementation, but may be implemented in the future (in
+which case they should be moved out of this section).
+
+This whole section should be considered <<[UNRESOLVED]>>.
+
+### Rollout of baseline-by-default for unlabeled namespaces
+
+If we wanted to change the default-default value from privileged to baseline, here is a possible
+conservative rollout path, potentially with multiple releases between steps. Steps could be dropped
+or combined for a more aggressive rollout:
+
+1. Admission plugin goes to GA
+2. Admission plugin enabled by default; Unlabeled namespaces treated as enforce=privileged,
+   audit=baseline, warn=privileged; Privileged pod (or PodTemplate controller) creation in an
+   unlabeled namespace triggers the namespace to be labeled as privileged
+3. Same as (2) but new namespaces are automatically labeled as unprivileged
+   (`security.kubernetes.io/privileged: false`), warn=baseline
+4. Default unlabeled namespaces to enforce=baseline
+
+Each step in the rollout could be overridden with a flag (e.g. force the admission plugin to step N)
+
+### PodSecurityPolicy Migration
```

Comment:
having a PSP migration plan seems worthwhile to hoist out of the "optional" section into beta or GA... though I'd put `UNRESOLVED` blocks around the specific plans for alpha

Reply:
specifically, the PSP timeline would dictate migration docs/tools/automation being available in 1.24